connect_ext_ppr/schemas.py

```diff
@@ -187,3 +205,11 @@ class TaskSchema(NonNullSchema):
     events: Events
     status: TasksStatusChoices
     error_message: Optional[str]
+
+
+class DeploymentRequestCreateSchema(NonNullSchema):
+    deployment: PrimaryKeyReference
```

Comment:
[COD-C] I though about such solution for the previous schemas. But in this case request will look like:
```
{
    ...
    "deployment": {
        "id": "DPL-13798"
    },
    ...
}
```
And I think that shorten way is more convenient:
```
{
    ...
    "deployment_id": "DPL-13798",
    ...
}
```
What do you think?